pubsub/pubsub.go

```diff
@@ -164,8 +164,14 @@ func (m *Message) Ack() {
 }
 
 // Nack tells the server that this Message was not processed and should be
```

Comment:
How about

   Nack (short for negative acknowledgment) is a ...

Also note the omitted "e" in acknowledgment; that is the preferred American spelling. https://grammarist.com/spelling/acknowledgment-acknowledgement

Reply:
Done.